book/src/lint_configuration.md

```diff
@@ -723,6 +723,16 @@ Minimum chars an ident can have, anything below or equal to this will be linted.
 * [`min_ident_chars`](https://rust-lang.github.io/rust-clippy/master/index.html#min_ident_chars)
 
 
+## `missing-docs-allow-unused`
+Whether to allow fields starting with underscore to skip documentation requirements
```

Comment:
nitpick: shouldn't it be "with underscores" or "with an underscore"? (repeated in other places)